src/winrt/activation/catalog.cpp

```diff
@@ -4,6 +4,10 @@
 #include <roapi.h>
 #include <winstring.h>
 #include <rometadataresolution.h>
+#include <combaseapi.h>
+#include <wrl.h>
```

Comment:
Ideally you should use something like C++/WinRT's `com_ptr` rather than WRL.

Reply:
This is shipping back to RS2, will that still work?  Scott made it sound like we couldn't use WinRT, but I may be confusing different concepts.

Reply:
If you're only depending on the Windows SDK here that may not work.